### PR TITLE
fix: Removed all references to Google firebase

### LINKED ISSUE #2195: Cannot create events in the mobile app because of Google notifications
**Describe the bug**
- Please work with @Ayush0Chaudhary for comments, advice and review of your PR.

Creating events fails with a `Notification Error` 

**To Reproduce**

1. Start app
2. Click on events
3. Add an event
4. Save event
5. Error is seen

**Expected behavior**
- There should be a no error

**Actual behavior**
- Error

**Screenshots**
-https://youtu.be/86AzK_Y_GCQ?t=429

**Additional details**
- We are no longer using the Firebase notifications feature and therefore this must be removed from the add event workflow. Non Firebase notifications is a GSoC 2024 draft idea
- You may need to coordinate your efforts with an API developer
- You must get the test **_code coverage up to 100%_** for any edited files. No functionality must be removed to achieve this goal

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Abhisheksainii**
**What kind of change does this PR introduce?**

BugFix

**Issue Number:**

Fixes #2195
#2303 

**Did you add tests for your changes?**
No.
P.S: Tests were already there, just commented those references which were discontinued.

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

No

**Summary**

The firebase notification services were discontinued as they were a dependency. They were causing errors while creating Events due to exception that was being thrown from backend ("Notification service not found"). The changes that I have made mostly includes commenting out and deleting the code.
Issue: https://github.com/PalisadoesFoundation/talawa/issues/2195

**Does this PR introduce a breaking change?**

Removed Firebase Notifications and its references, the impact has been taken care of in this issue (https://github.com/PalisadoesFoundation/talawa-api/issues/1500) 

**Other information**
Merge this PR after this (https://github.com/PalisadoesFoundation/talawa-api/pull/1503) has been merged by backend, otherwise it will cause issue while creating events.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Abhisheksainii** [comment]
@palisadoes @noman2002 can someone please approve the workflows

**Review by Kevoniat — APPROVED** [review]

**Comment by palisadoes** [comment]
@Abhisheksainii

1. Have you tested this with the latest upstream of the API? 
2. We just merged a PR that removed firebase.

**Comment by Abhisheksainii** [comment]
> @Abhisheksainii
> 
> 1. Have you tested this with the latest upstream of the API?
> 2. We just merged a PR that removed firebase.

I haven't but will test it.

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart [hunk 1]
```diff
@@ -87,7 +89,44 @@ class ProfilePageViewModel extends BaseModel {
   /// * `Future<void>`: Resolves when user logout
   Future<void> logout(BuildContext context) async {
     // push custom alert dialog with the confirmation message.
-    navigationService.pushDialog(logoutDialog());
+    navigationService.pushDialog(
```
Comment (literalEval) [inline lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:92]: Why is logout dialog not called as before ? What is the motto behind calling it directly here ? ViewModel should not have code for View.

File: lib/firebase_options.dart [hunk 2]
```diff
@@ -1,93 +1,93 @@
-// File generated by FlutterFire CLI.
+// // File generated by FlutterFire CLI.
```
Comment (literalEval) [inline lib/firebase_options.dart:1]: Remove the lines/files if not needed. Please don't comment and leave as it is.

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -57,9 +57,9 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
-        <meta-data
+        <!-- <meta-data
```
Comment (literalEval) [inline android/app/src/main/AndroidManifest.xml:60]: Remove this tag.

File: lib/services/database_mutation_functions.dart [hunk 4]
```diff
@@ -191,18 +191,18 @@ class DataBaseMutationFunctions {
...
-          WidgetsBinding.instance.addPostFrameCallback(
-            (_) => navigationService.showTalawaErrorDialog(
-              "Notification Feature is not installed",
-              MessageType.error,
-            ),
-          );
-        }
-        return false;
-      }
+      // if (exception.graphqlErrors[i].message ==
```
Comment (literalEval) [inline lib/services/database_mutation_functions.dart:194]: Remove.

**Review by literalEval — CHANGES_REQUESTED** [review]

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart (hunk 1 above)
Comment (Abhisheksainii) [reply lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:92]: This change was done by @sumitra19jha 2 years back, due to formatting and linting it has been shown here.
The profile view model contains similar functions like this.

**Comment by Abhisheksainii** [comment]
Don't merge this PR, some tests are left

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart (hunk 1 above)
Comment (literalEval) [reply lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:92]: If so, please open an issue for making this change. Comment the issue number in this PR's itself.

**Comment by Cioppolo14** [comment]
@Abhisheksainii Are you still working on this?

**Comment by Abhisheksainii** [comment]
Yes

**Comment by Abhisheksainii** [comment]
Yes, testing phase is going on.
The issue is from backend side due to which I am unable to test.

**Comment by Abhisheksainii** [comment]
@Kevoniat @Cioppolo14 you can merge the PR, testing has been done

File: test/utils_tests/queries_test.dart [hunk 5]
```diff
@@ -27,16 +27,16 @@ void main() {
...
-      var mutation = false;
-      expect(mutation, false);
-
-      final fnData = Queries().saveFcmToken('token123');
-      if (fnData.contains('token123')) {
-        mutation = true;
-      }
-      expect(mutation, true);
-    });
+    // test("Check if saveFcmToken works correctly", () {
```
Comment (literalEval) [inline test/utils_tests/queries_test.dart:30]: Remove these tests.

**Review by literalEval — CHANGES_REQUESTED** [review]

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart (hunk 1 above)
Comment (literalEval) [reply lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:92]: Did you open the respective issue ?

File: test/utils_tests/queries_test.dart (hunk 5 above)
Comment (Abhisheksainii) [reply test/utils_tests/queries_test.dart:30]: okay

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart (hunk 1 above)
Comment (Abhisheksainii) [reply lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:92]: yes

**Comment by Abhisheksainii** [comment]
@literalEval I have done the changes

**Comment by Abhisheksainii** [comment]
@literalEval all the required changes has been done and issue has been also created

**Comment by Abhisheksainii** [comment]
@literalEval @Kevoniat kindly merge the PR as all the required changes has been done

**Comment by palisadoes** [comment]
Please work with @literalEval @Kevoniat for approvals. I will not be checking back in.

**Comment by Abhisheksainii** [comment]
@Kevoniat @literalEval can you please approve the workflows?

**Comment by Azad99-9** [comment]
@Abhisheksainii please write tests to the missing lines.

**Comment by Abhisheksainii** [comment]
> @Abhisheksainii please write tests to the missing lines.

I think the files that I have changed doesn't require tests to be written and as for the the issue in profile_page_view_model.dart file. It was assigned to Parag and it has been merged.

**Comment by palisadoes** [comment]
@Kevoniat @DMills27 @Ayush0Chaudhary @literalEval please take a look. We really need this code removed so we can proceed with our roadmap

**Review by literalEval — APPROVED** [review]
LGTM

**Review by palisadoes — APPROVED** [review]

**Pull request merged**